src/frontend/styles/vars.pcss

```diff
@@ -3,10 +3,22 @@
   --color-text-second: #5d6068;
   --color-line-gray: #E8E8EB;
   --color-link-active: #2071cc;
-  --color-button-danger: #ff5159;
   --color-bg-light: #f8f7fa;
   --color-page-active: #ff1767;
 
+  --color-button-primary: #3389FF;
+  --color-button-primary-hover: #2E7AE6;
+  --color-button-primary-active: #296DCC;
+
+  --color-button-secondary: #F3F6F8;
```

Comment:
btw mockups did not contain grey button example, so, i picked random grey shade. @neSpecc can you please verify the color?